### PR TITLE
Term entry pandas pop py 4522

### LINKED ISSUE #4522: [Term Entry] Python:Pandas dataframe pop
### Reason/inspiration (optional)

"We would like a new term entry in the `dataframe` concept for [Pandas](https://www.codecademy.com/resources/docs/pandas): pop The entry should go in a new file under `docs/content/pandas/concepts/dataframe/terms/pop/pop.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use the trace function to find the trace of a square matrix.
- Should contain information for functions and syntax.
- Should have a Python codebyte with a working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by JenHunnisett**
### Description

This is a new term entry for the '.pop()' method for the dataframe concept within Pandas. The entry is in a new file under docs/content/pandas/concepts/dataframe/terms/pop/pop.md

The entry describes the term and how it is used. I've also included:

A description of the method
A ## Syntax section that includes the syntax of the method  and its details
An ## Example section that includes a python codeblock and shell that shows the example output.
A ## Codebyte Example that includes a codebyte/python runnable codeblock

### Issue Solved
Closes #4522 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/pop/pop.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/pop/pop.md:14]:
```suggestion
---
Title: '.pop()'
Description: 'Removes and returns a specified column from a DataFrame.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Web Development'
Tags:
  - 'Data Structures'
  - 'Methods'
  - 'Pandas'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---
```

File: content/pandas/concepts/dataframe/terms/pop/pop.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+The **`.pop()`** method removes a single specified column from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object. The original `DataFrame` object is modifed, and the removed column is returned as a Pandas Series object.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/pop/pop.md:16]:
```suggestion
In Pandas, the **`.pop()`** method removes a single specified column from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object. The original `DataFrame` object is modified, and the removed column is returned as a pandas Series object.
```

File: content/pandas/concepts/dataframe/terms/pop/pop.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Syntax
+
+```pseudo
+DataFrame.pop(column_label)
+```
+
+`DataFrame.pop()` takes a single parameter, the label of the column to be removed and returned.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/pop/pop.md:24]:
```suggestion
- `DataFrame`: The DataFrame object from which a column is removed.
- `column_label`: The label of the column to remove. This should be a string representing the column name.
```

File: content/pandas/concepts/dataframe/terms/pop/pop.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+3      Star
+Name: col 3, dtype: object
+
+DataFrame after pop:
+    col 1   col 2      col 4
+0      1       Red        Sweet
+1      2       Blue       Sour
+2      3       Green      Bitter
+3      4       Pink       Salty
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/pop/pop.md:66]: Add a line below this to explain the above code, example:

The code above creates a pandas DataFrame from a dictionary, prints the original DataFrame, uses the `.pop()` method to remove the column 'col 3' and return it as a Series, and then prints the popped data and the modified DataFrame.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @JenHunnisett! Thank you for contributing to the Codecademy Docs! The entry is well written!

I've requested a few changes in the entry, could you please review the suggestions and modify them at your earliest convenience? Feel free to discuss any concerns 😄 
Thank you! 🚀

**Comment by JenHunnisett** [comment]
@mamtawardhani Thanks for your helpful suggestions, I have incorporated them, so whenever you're ready please take another look and let me know if I'm good to go :-)

**Comment by JenHunnisett** [comment]
@letsbuildanything quick query, this is my 2nd pull request and now that the 1st one has just been merged I can see that I'm credited as an "Anonymous Contributor" when I thought I'd setup my profile correctly to allow me to be credited by my profile name for the contribution, so I'm wondering if you can tell me what I've missed? Thankyou!

**Comment by letsbuildanything** [comment]
Hi @JenHunnisett,
Below Instruction may help you to solve your query. Let me know if you need any further assistance!
> It’s likely your GitHub username hasn’t been added to your Codecademy account yet. You can add your GitHub username in your Profile settings on Codecademy. After adding your GitHub username, logout of Codecademy and log back in using the GitHub login method. Be sure that the email address associated with your Codecademy account is listed as a primary email in your [GitHub settings](https://github.com/settings/emails).

**Comment by JenHunnisett** [comment]
Thankyou @letsbuildanything :-) much appreciated!

**Pull request merged**